lib/src/boilerplate_suggestors/constants.dart

```diff
@@ -1,2 +1,2 @@
-const reactVersionRange = '^5.4.0-alpha';
-const overReactVersionRange = '^3.5.0-alpha';
+const overReactVersionRange = '^3.5.0';
+const overReactTestVersionRange = '^3.9.0';
```

Comment:
Whoops, wrong version!
```suggestion
const overReactTestVersionRange = '^2.9.0';
```